stage/pages/lms/course_page_lms.py

```diff
@@ -0,0 +1,9 @@
+from edxapp_pages.lms.course_info import CourseInfoPage
```

Comment:
Makes sense, but should this be added to edx-platform (sooner or later)? It seems like a non-intrusive change.